db/seeds.rb

```diff
@@ -29,7 +29,7 @@ def rugged_repo
   end
 
   p "Adding Track: #{track_slug}"
-  track = Track.create!(slug: track_slug, title: track_slug, repo_url: v3_url)
+  track = Track.create!(slug: track_slug, title: track_slug.titleize, repo_url: v3_url)
```

Comment:
The Track slug -> Name mapping is not always correct. Since you have the data, don't you want to read this from the config file and use this as fallback only?

Reply:
Yeah. This seeds is only temporary. Lots of the V3 repo data is very messy right now.

The actual db will be some sort of ETL from v2

Reply:
Nice.